packages/hash/frontend/src/pages/[shortname]/entities/[entity-uuid].page/entity-editor/links-section/link-table/cells/linked-with-cell/entity-selector.tsx

```diff
@@ -38,14 +38,17 @@ export const EntitySelector = ({
         setLoading(true);
         const { data } = await aggregateEntities({
           data: {
-            rootEntityTypeIds: expectedEntityTypes.map(
-              ({ schema }) => schema.$id,
-            ),
+            operation: {
+              /** @todo - we want more than one entity type here */
+              entityTypeId: expectedEntityTypes.map(
+                ({ schema }) => schema.$id,
+              )[0],
+            },
```

Comment:
The Aggregation syntax doesn't take multiple right now, I'm not sure what to do in this PR. We could replace this with an unconstrained aggregateEntities call, but I'm not sure if the logic using this data relies on the roots being filtered already. We could split it into multiple queries, but that's temporary hacky investment I'm trying to avoid.

Reply:
can we make it an array?

Reply:
How come `entityTypeId` is a standalone field and not just a `filter`? We'd get multiple constrained entity types (Using `OR`) if we allow it as just a filter and remove this explicit field

Reply:
Yes – I think ultimately we just want it to be part of the filters, along with timestamps etc. I think there was some issue with doing this though that we needed to address (those fields being split across multiple tables?)

Reply:
> I think there was some issue with doing this though that we needed to address (those fields being split across multiple tables?)

I think the main thing we need to address is just taking an array of `field`s instead of a single one in the aggregation syntax. Once we do that we can conditionally match on the contents of the fields to identify if something is in `properties` or not.